docs/specification.md

```diff
@@ -194,6 +194,7 @@ The list of erros may be inaccurate and incomplete, it will be updated during th
 | `NOT_PENDING_ADMINISTRATOR`    | Authorized sender is not the current pending administrator   |
```

Comment:
Don't forget to update `NOT_ADMINISTRATOR` above.

Reply:
I've updated this.